src/scss/includes/components/closeButton.scss

```diff
@@ -0,0 +1,21 @@
+.closeButton {
+  height: 35px;
+  width: 35px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 22px;
+  cursor: pointer;
+  position: relative;
+  top: -9px;
```

Comment:
I think it's best if CloseButton totally ignores where it will be positioned

Reply:
For me it's not really positionning stuff but a shift to ensure the cross is visually aligned with the surrounding content but the actual clickable size is larger. So this is better if it's not managed by the calling sites.
But I understand your point. I'll try doing it with margins instead.